overlay/files-ubuntu-arm-rpi/system/oem/11_persistency.yaml

```diff
@@ -0,0 +1,37 @@
+name: "Configure persistent dirs bind-mounts"
```

Comment:
dont we have this already on the generic overlay files?

Reply:
tried with the generic one and it seems to be working well so removing the rpi one